packages/office-ui-fabric-react/src/components/Image/Image.tsx

```diff
@@ -127,7 +127,7 @@ export class Image extends BaseComponent<IImageProps, IImageState> {
   private _evaluateImage(): boolean {
     let { src } = this.props;
     let { loadState } = this.state;
-    let isLoaded = (src && this._imageElement.naturalWidth > 0 && this._imageElement.naturalHeight > 0);
+    let isLoaded = (src && this._imageElement.complete);
```

Comment:
In the SVG scenario is `naturalWidth` actually 0 or NaN or undefined? Can we rely on something here to determine successful loading vs failed loading?